src/ops.c

```diff
@@ -229,11 +229,11 @@ shift_line(
     int	amount,
     int call_changed_bytes)	// call changed_bytes()
 {
-    int		count;
+    long long	count;
     int		i, j;
     int		sw_val = (int)get_sw_value_indent(curbuf);
 
-    count = get_indent();	// get current indent
+    count = (long)get_indent();	// get current indent
```

Comment:
```suggestion
    count = (long long)get_indent();	// get current indent
```
Converting to a larger type will be done automatically, so the cast may not be needed. However, if you want to cast it explicitly, I think this should be `(long long)`.

Reply:
ah yes, I missed it when switching to (long long). Will fix